docs/api-reference/react/deckgl.md

```diff
@@ -92,6 +80,27 @@ Current pitch - used to define a mercator projection if `viewState` is not suppl
 
 The following semantics of the standard React `children` property are considered experimental.
 
+#### Render callbacks
```

Comment:
To me it seems like this section should be moved down two sections. Should be under "Position Children in Viewport" since it shows a more advanced scenario relating to that topic?